taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+
+app_name = "taxi"
+urlpatterns = [
+    path("", views.index, name="index"),
+]

```

Comment:
```suggestion
from taxi.views import index


app_name = "taxi"
urlpatterns = [
    path("", index, name="index")
]
```
use absolute import instead of relative